<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flipigas - Administrador</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>351</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="InformacionTab">
     <attribute name="title">
      <string>Informacion</string>
     </attribute>
     <widget class="QLabel" name="InfoGeneralLabel">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>371</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Información General</string>
      </property>
     </widget>
     <widget class="QFrame" name="InformacionFrame">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>521</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="TiempoEntregaLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>201</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Tiempo estimado de entrega:</string>
       </property>
      </widget>
      <widget class="QLabel" name="CantidadSolicitudesActLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>221</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Cantidad de solicitudes actuales:</string>
       </property>
      </widget>
      <widget class="QLabel" name="CantidadSolicitudesAttLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>231</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Cantidad de solicitudes atendidas:</string>
       </property>
      </widget>
      <widget class="QLabel" name="TiempoDisponibilidadCamionLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>311</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Tiempo de disponibilidad del siguiente camión:</string>
       </property>
      </widget>
      <widget class="QLabel" name="TiempoEntrega">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>20</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>XYZ</string>
       </property>
      </widget>
      <widget class="QLabel" name="CantidadSolicitudesAct">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>50</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>XXXXXX</string>
       </property>
      </widget>
      <widget class="QLabel" name="CantidadSolicitudesAtt">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>80</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>YYYYYY</string>
       </property>
      </widget>
      <widget class="QLabel" name="TiempoDisponibilidadCamion">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>110</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>XYZ</string>
       </property>
      </widget>
      <widget class="QLabel" name="minutosLabel1">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>minutos.</string>
       </property>
      </widget>
      <widget class="QLabel" name="minutosLabel2">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>110</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>minutos.</string>
       </property>
      </widget>
      <widget class="QPushButton" name="VolverAdminButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>170</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Volver al menu anterior</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="InfoGeneralIcon">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Imgs/home-512.gif</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="RepartidorTab">
     <attribute name="title">
      <string>Repartidor</string>
     </attribute>
     <widget class="QFrame" name="RepartidorFrame">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>601</width>
        <height>101</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="CamionesEnRepartoLabel">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>10</y>
         <width>371</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>22</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Camiones en reparto</string>
       </property>
      </widget>
      <widget class="QLabel" name="FiltrarporLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>81</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Filtrar por:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="TipoCamionFilterBox">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>70</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>Tipo de camión</string>
       </property>
       <item>
        <property name="text">
         <string>Tipo de camión</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="TipoCamionSortBox">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>70</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>Tipo de camión</string>
       </property>
       <item>
        <property name="text">
         <string>Tipo de camión</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="OrdenarporLabel">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>70</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Ordenar por:</string>
       </property>
      </widget>
      <widget class="QLabel" name="CamionesEnRepartoIcon">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>Imgs/191.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>601</width>
        <height>181</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </widget>
    <widget class="QWidget" name="VentasyBodegaTab">
     <attribute name="title">
      <string>Ventas y Bodega</string>
     </attribute>
     <widget class="QLabel" name="InfoVentasyBodegaLabel">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>401</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Información de ventas y bodega</string>
      </property>
     </widget>
     <widget class="QLabel" name="InfoventasIcon">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>Imgs/meter_calculator_11440.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QFrame" name="VentasFrame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>351</width>
        <height>131</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="VentasLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ventas</string>
       </property>
      </widget>
      <widget class="QLabel" name="ClasificarLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Clasificar por:</string>
       </property>
      </widget>
      <widget class="QLabel" name="CerroLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Cerro:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="CerroBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>80</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Escoja cerro...</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="TipoLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>58</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Tipo:</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="CataliticoRadioButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>110</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Catalítico</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="NoCataliticoRadioButton">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>110</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>No Catalítico</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="AmbosRadioButton">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>110</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ambos</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="BodegaFrame">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>100</y>
        <width>221</width>
        <height>211</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="BodegaLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Bodega</string>
       </property>
      </widget>
      <widget class="QLabel" name="CilindrosRestantesLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Cilindros restantes:</string>
       </property>
      </widget>
      <widget class="QLabel" name="CilindrosRestantes">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>XYZWY</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SolicitarReposicionButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>171</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Solicitar Reposición</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>151</width>
        <height>91</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QRadioButton" name="CincoLTRadioButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="DiezLTRadioButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="QuinceLTRadioButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>51</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>15</string>
       </property>
      </widget>
      <widget class="QLabel" name="TamanoLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Tamaño (en litros):</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="CalcularVentasButton">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>250</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Calcular ventas</string>
      </property>
     </widget>
     <widget class="QLabel" name="Ventas">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>70</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>12345678</string>
      </property>
     </widget>
     <widget class="QLabel" name="SignoPesoLabel">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>65</y>
        <width>16</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>$</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
